src/MongoDB.Driver/PipelineStageDefinitionBuilder.cs

```diff
@@ -927,6 +927,108 @@ public static class PipelineStageDefinitionBuilder
             return Match(new ExpressionFilterDefinition<TInput>(filter));
         }
 
+        /// <summary>
+        /// Creates a $merge stage.
+        /// </summary>
+        /// <typeparam name="TInput">The type of the input documents.</typeparam>
+        /// <typeparam name="TOutput">The type of the output documents.</typeparam>
+        /// <param name="outputCollection">The output collection.</param>
+        /// <param name="mergeOptions">The merge options.</param>
+        /// <returns>The stage.</returns>
+        public static PipelineStageDefinition<TInput, TOutput> Merge<TInput, TOutput>(
+            IMongoCollection<TOutput> outputCollection,
```

Comment:
Does this need to be an IMongoCollection or can it should it be a MongoNamespace (i.e. a class that just has db/coll names and not all the behavior of IMongoCollection)

Reply:
Maybe it could be be a CollectionNamespace, but all other methods in this type that take a collection already take an IMongoCollection.